docs/feature_audio.md

```diff
@@ -1,21 +1,86 @@
 # Audio
 
-Your keyboard can make sounds! If you've got a Planck, Preonic, or basically any AVR keyboard that allows access to certain PWM-capable pins, you can hook up a simple speaker and make it beep. You can use those beeps to indicate layer transitions, modifiers, special keys, or just to play some funky 8bit tunes.
+Your keyboard can make sounds! If you've got a spare pin you can hook up a simple speaker and make it beep. You can use those beeps to indicate layer transitions, modifiers, special keys, or just to play some funky 8bit tunes.
 
-Up to two simultaneous audio voices are supported, one driven by timer 1 and another driven by timer 3.  The following pins can be defined as audio outputs in config.h:
+To activate this feature, add `AUDIO_ENABLE = yes` to your `rules.mk`.
 
-Timer 1:
-`#define B5_AUDIO`
-`#define B6_AUDIO`
-`#define B7_AUDIO`
+## AVR based boards
+Up to two simultaneous tones can be rendered. with one speaker driven by timer 1 and another driven by timer 3.  The following pins can be configured as audio outputs in `config.h`:
 
-Timer 3:
-`#define C4_AUDIO`
-`#define C5_AUDIO`
-`#define C6_AUDIO`
+for the primary speaker, with Timer 3, pick ONE these pins:
+`#define AUDIO_PIN_C4`
+`#define AUDIO_PIN_C5`
+`#define AUDIO_PIN_C6`
 
-If you add `AUDIO_ENABLE = yes` to your `rules.mk`, there's a couple different sounds that will automatically be enabled without any other configuration:
+and *optionally*, a secondary speaker, using Timer 1, on ONE of these pins:
```

Comment:
I'm not a huge fan of the wording here.   You can use Timer 1 (B pins) as the primary pin.  This is common for Elite C controllers, as it exposes a number of B pins that aren't used. 

Additionally, some controllers don't have the timer 3, IIRC.

Reply:
since this is in the AVR specific part, how about i mention Atmega32U4 as the MCU?
this might make things clearer/specific

Reply:
3223faa reworks the define names a bit to also include/handle AUDIO_PIN_Bx as only/primary output - plus updated documentation.
better this way?

Reply:
It's better, but I'm still not exactly happy with it.  

I think that is because, at least for the atmega32u4, you can use either timer 1 or timer 3, and that neither should be considered primary/secondory. Unless there is a good reason (aside from having both connected). 

As far as I'm aware, both sets are completely valid and there is no issues with using either.

Reply:
since zvecr also is in favor of `AUDIO_PIN Bx` for AVR, i reverted only those and updated the documentation accordingly - how is this:  3566defcfb3a852b77fb5b11106ecae0d09f61fc ?

Reply:
marking this sub-thread as resolved - changes are pushed and need another review...